handshake_ix.go

```diff
@@ -167,12 +170,11 @@ func ixHandshakeStage1(f *Interface, addr netip.AddrPort, via *ViaSender, packet
 		if rc == nil {
 			f.l.WithError(err).WithField("udpAddr", addr).
 				WithField("handshake", m{"stage": 1, "style": "ix_psk0"}).WithField("cert", remoteCert).
-				Info("Unable to handshake with host due to missing certificate version")
-			return
+				Info("Might be unable to handshake with host due to missing certificate version")
```

Comment:
This seems like it could get quite spammy and its not directly obvious if there is any action the operator can take to fix it or if they even need to. Think we can take this to debug level and let the initiator deal with logging the error on stage 2